mixes/reducer.js

```diff
@@ -60,15 +69,26 @@ function createReducer (config) {
     [loadMixEnd]: (state, action) => ({
       ...state, isLoading: false
     }),
-    [saveMix]: (state, action) => loop({
-      ...state, isSaving: true
-    }, Effects.batch([
-      // currently, the component passes a nestedMix into this saveMix action.payload
-      // maybe in the future we should pass mixId and nestMix with channels, clips
-      Effects.promise(runSaveMix, action.payload),
-      Effects.constant(saveMixEnd())
-    ])),
-    [saveMixSuccess]: (state, action) => state,
+    [saveMix]: (state, action) => {
+      const nestedMix = action.payload;
+      const { meta: mixMeta, primaryTracks } = nestedMix
+
+      // TODO: save primaryTracks and sampleTracks sample meta here too
+
+      const effects = Effects.batch([
+        Effects.promise(runSaveMix, pick(nestedMix, 'id', 'channel')),
+
+        // TODO: should saveMeta only trigger in saveMixSuccess?
+        Effects.constant(saveMeta(mixMeta)),
+        Effects.constant(saveMixEnd())
+      ])
+
+      return loop({
+        ...state, isSaving: true
+      }, effects)
+    },
+    [saveMixSuccess]: (state, action) => loop(state,
+      Effects.constant(loadMixList())), // TODO: is this necessary?
```

Comment:
so... if i dont add this line, any changes we made to the mix wont be updated in the store when we save. see question in PR desc for more context (does the store always get updated, eg when we edit mix title? or does that happen only on save?)